### DISCUSSION TITLE
How to access levels?

### DISCUSSION THREAD

**Discussion by baranacikgoz** [mapsforge/mapsforge#1265] [Q&A] [answered]
Your example app has some maps like Chemnitz University Indoor etc. And there are buttons to switch between these levels. I have a building map too and it has levels but how can i access these levels with mapsforge and create buttons that knows the levels?

**Comment by devemux86** [comment]
Which example? Or do you mean jeo (indoor maps) in [VTM](https://github.com/mapsforge/vtm) map library?

There can be layers in themes, you can check the [render theme documentation](https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md#stylemenus).

**Reply by baranacikgoz** [reply]
This [flutter_mapsforge Example](https://github.com/mikes222/mapsforge_flutter/tree/master/example). There are sections like this; 
`new MapFileData(
    url: "https://raw.githubusercontent.com/mikes222/mapsforge_flutter/master/example/map_files/Chemnitz Uni.map",
    fileName: "Chemnitz Uni.map",
    displayedName: "Chemnitz - University (Indoor)",
    initialPositionLat: 50.81348,
    initialPositionLong: 12.92936,
    initialZoomLevel: 18,
    indoorZoomOverlay: true,
    indoorLevels: {1: 'OG', 0: 'EG', -1: 'UG'},
  ),
`
How `indoorLevels` code gets the levels? What makes my map's level's readable to  `indoorLevels` cod? Tagging objects with josm indoorHelper plugin like `level=0` or adding different layers?

**Comment by devemux86** [comment, marked as answer]
You should ask in that repository how they use the map files / render themes.
